README.md

```diff
@@ -0,0 +1,293 @@
+> Mind you that it's an experimental API
+
+It's even meant for the browser - so no need to do any transformation at all??? just provide the map and you don't even need rollup while developing? sounds crazy? let's try it out :hugs:
+
+It currently only works in chrome 75+ and you need to enable a flag.
+So enter `chrome://flags/` in the URL bar and then search for `Built-in module infra and import maps` and enable it.
+Here is a direct link to it: [chrome://flags/#enable-built-in-module-infra](chrome://flags/#enable-built-in-module-infra).
+
+In order to use it with a browser let's create an `index.html` file.
+```html
+<html lang="en-GB">
+<head>
+  <script type="importmap">
+    {
+      "imports": {
+        "lit-html": "./node_modules/lit-html/lit-html.js",
+        "lit-html/": "./node_modules/lit-html/",
+        "lit-element": "./node_modules/lit-element/lit-element.js",
+        "lit-element/": "./node_modules/lit-element/"
+      }
+    }
+  </script>
+  <title>My app</title>
+</head>
+
+<body>
+  <script type="module" src="./main.js"></script>
+</body>
+
+</html>
+```
+
+then serve it via `npx http-server` and it just works.
+What kind of black magic is this? no build step and I can still keep writing bare modules?
+
+Lets break it down:
+```js
+import { html } from 'lit-html';
+// will actually import "./node_modules/lit-html/lit-html.js"
+// because of
+// "lit-html": "./node_modules/lit-html/lit-html.js",
+
+import { repeat } from 'lit-html/directives/repeat.js'
+// will actually import "./node_modules/lit-html/directives/repeat.js"
+// beacause of
+// "lit-html/": "./node_modules/lit-html/",
+```
+
+So this means
+1. you can import the bare import directly as it is mapped to a specific file
+2. you can import subfolders and subfiles as a bare module + '/' as it is mapped to a folder
+3. you can NOT omit the `.js` when importing a subfolder/subfile
+
+
+#### What does that mean for my production build?
+
+You still want to be able to do optimized production builds.
+The only thing is that you probably want to replace the `rollup-plugin-node-resolve` with something that respects your `import map` instead of using the node resolve.
+
+And actually really nice would be if you could just point to your `index.html` and rollup should figure out what are your entry points and if there is an import map.
+
+Luckily we got you covered with `rollup-plugin-index-html`.
+
+So let's install it
+```bash
+yarn add --dev rollup-plugin-index-html
+```
+
+and adjust our `rollup.config.js`
+```js
+import indexHTML from "rollup-plugin-index-html";
+
+export default config => ({
+  input: "./index.html",
+  output: {
+    dir: "dist", // replaced file as we now output at least one index.html and one js file
+    format: "esm" // replaced iife as we also generate an index.html which then loads the es module bundle
+  },
+  plugins: [indexHTML({ ...config, rootDir: __dirname })]
```

Comment:
```suggestion
  plugins: [indexHTML(config)]
```

Reply:
this wasn't needed, right?

Reply:
jup too much copy/paste today 🙈  will change